Week_01_Basics/homework-portfolio/style.css

```diff
@@ -0,0 +1,100 @@
+nav {
+	text-align: center;
+	color: #707070;
+	font-size: 14pt;
+	font-style: bold;
+	font-family: sans-serif;
+	color: #23635a;
+	padding-left: 50px;
+}
+body {
+	text-align: left;
+	vertical-align: baseline;
+	background: url(./bg.png);
```

Comment:
great use for background.  The proper css for this is

```background-image: url(./bg.png);```

The browser you're using is lenient so it will fix this up for you automatically.